course_info/templates/course_info/partials/details.html

```diff
@@ -0,0 +1,211 @@
+{% verbatim %}
+
+<nav>
+    <h3 class="breadcrumbs">
+        {% endverbatim %}
+        <a href="{% url 'dashboard_account' %}">Admin Console</a>
+        &gt; <a href="#/">Find Course Info</a>
+        {% verbatim %}
+        <small><i class="fa fa-chevron-right"></i></small>
+        Details for {{courseInstance.title}}
+    </h3>
+</nav>
+
+<main class="float=left col=60">
```

Comment:
This class looks weird to me, is it supposed to have these `=` signs in it and whatnot?